pkg/apis/sources/v1alpha1/githubsource_types.go

```diff
@@ -88,14 +88,23 @@ type SecretValueFromSource struct {
 }
 
 const (
-	// gitHubSourceEventPrefix is what all GitHub event types get
+	// gitHubEventTypePrefix is what all GitHub event types get
 	// prefixed with when converting to CloudEvents.
-	gitHubSourceEventPrefix = "dev.knative.source.github"
+	gitHubEventTypePrefix = "dev.knative.source.github"
+
+	// gitHubEventSourcePrefix is what all GitHub event sources get
+	// prefixed with when converting to CloudEvents.
+	gitHubEventSourcePrefix = "//github.com"
 )
 
-// GetGitHubSourceEventType returns the GitHub CloudEvent type value.
-func GetGitHubSourceEventType(ghEventType string) string {
```

Comment:
We agreed in the Registry discussions to just use `github.com` as the CE source. 
IMO that is not useful, we need more information there. I'm therefore adding the owner and the repo as well.
Example: `//github.com/nachocano/eventing`. 
Should we use the starting slashes //?